src/main/java/gomedic/logic/commands/editcommand/EditActivityCommand.java

```diff
@@ -80,6 +81,10 @@ private static Activity createEditedActivity(Activity activityToEdit,
         Title title = editActivityDescriptor.getTitle().orElse(activityToEdit.getTitle());
         Description description = editActivityDescriptor.getDescription().orElse(activityToEdit.getDescription());
 
+        if (activityToEdit.isAppointment()) {
+            PatientId patientId = editActivityDescriptor.getPatientId().orElse(activityToEdit.getPatientId());
```

Comment:
Actually, there is no way to edit the patient id, you can just have `patientId = activityToEdit.getPatientId()`, and also you don't need to have the edit Activity Descriptor

Reply:
lol oops ye ill remove that